docs/contributing-testing.rst

```diff
@@ -24,6 +24,57 @@ Put your code's tests in the appropriate subdirectory of the ``tests`` directory
 If your feature is substantially new, you may decide to create a new ``tests`` subdirectory in the interest
 of code organization.
 
+How do I add a new test suite?
```

Comment:
I'd suggest moving this section to the bottom of this document, because I expect that adding a new test suite will be a less common workflow than running tests. My intent with this document is to order it according to a typical contribution onboarding path.

Reply:
Good catch, should've been addressed now.